frontend/app/services/activity-service.js

```diff
@@ -26,19 +26,23 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
-module.exports = function(HALAPIResource, $http, PathHelper){
+module.exports = function(
+  HALAPIResource,
+  $http,
+  I18n,
+  NotificationsService,
+  ConfigurationService,
+  $timeout,
+  PathHelper){
```

Comment:
I'm going to mark your comments with a :ballot_box_with_check: to mark them as 'addressed' for myself.

:ballot_box_with_check: